config/secrets.yml

```diff
@@ -1,7 +1,14 @@
 development:
+  primary_key: 5iYPUMb6NWxlsyiRGyBrVgfY6ZPhm5ZA
+  key_derivation_salt: q8sHPMCYthvtuM5N7mv2438my5v4DPaP
   secret_key_base: 101615e3369d108c13f7182caf9bb988fc8f2d8d309ebd16d39b34bece4b4bd0944df576bfa2ff35984e7c447658cc25810540b50759c15c2b94f8ef26867a8a
 test:
   secret_key_base: 3fb6b8dc769442c5f268a1fc6d1238e80bd6cf07b240d02ae622bd635a79f19342363291f053c80ac5f0132fe773f73e561022a46324a1310d9f0d368414d369
+  primary_key: 5iYPUMb6NWxlsyiRGyBrVgfY6ZPhm5ZA
+  key_derivation_salt: q8sHPMCYthvtuM5N7mv2438my5v4DPaP
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   notify_api_key: <%= ENV["GOVUK_NOTIFY_API_KEY"] %>
+  primary_key: <%= ENV["ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY"] %>
+  key_derivation_salt: <%= ENV["ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT"] %>
+  
```

Comment:
Not sure quite what's going on with the line ending on this file, but looks like it needs a fix.